### ISSUE TITLE
Genome Browser: There aren't any instructions on how to use / control the interface. 

### ISSUE THREAD

**Issue by leehart** [wtchg-kwiatkowski/observatory-web#190]
HOME [tab] > Explore genomic data through the Viewer [heading] > View genome [button]

The user will have to discover (or be taught) the drag-to-scroll feature, the scroll-wheel-to-zoom feature, etc.

(Maybe provide a link to some documentation. Maybe write some user-guide documentation!)

Ideally the controls would be intuitive, but do we have any evidence or reassurance?

**Comment by benjeffery** [comment]
A translucent overlay that fades on first interaction would prob work best

**Comment by leehart** [comment]
Yes, I like the idea of coupling the user-guide with the user-interface, instead of maintaining a completely separate thing.

Ideally the user should be able to reinstate the overlays / in-app help / doc features at any time.
